src/sidebar/store/test/util-test.js

```diff
@@ -36,6 +51,16 @@ describe('reducer utils', function() {
   });
 
   describe('#createReducer', function() {
+    it('returns an object if input state is undefined', function() {
+      // See redux.js:assertReducerShape in the "redux" package.
+      const reducer = util.createReducer(fixtures.update);
+      const initialState = reducer(undefined, {
+        type: 'ADD_ANNOTATIONS',
```

Comment:
I think it might be clearer here to name this `DUMMY_ACTION` or something to indicate that it is an action that the reducer doesn't handle. Internally Redux will dispatch actions with [randomized "type" strings](https://github.com/reduxjs/redux/blob/9c9a4d2a1c62c9dbddcbb05488f8bd77d24c81de/src/utils/actionTypes.js#L16) for various purposes.